proto/xudrpc.proto

```diff
@@ -83,6 +83,13 @@ service Xud {
     };
   }
 
+  /* Opens a payment channel given pub key, amount and currency. */
```

Comment:
How about "Opens a payment channel to a peer with the given node pub key for the specified amount and currency."